<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="ManualYAxisControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>183</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>183</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>183</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Manual Y Axis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="auto_rescale">
      <property name="text">
       <string>AUTO RESCALE</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Y max</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="y_max_value"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y min</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="y_min_value"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>OR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>y_min_value</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>y_min_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_max_value</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>y_max_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_rescale</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>y_auto_rescale_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>y_min_event()</slot>
  <slot>y_max_event()</slot>
  <slot>y_auto_rescale_event()</slot>
 </slots>
</ui>
